_layouts/downloads.html

```diff
@@ -403,15 +403,15 @@ <h5 class="section-toggle section-toggle-start-open ">Tools for Developers</h5>
                       <div class="component-releases-structure theme-darkgoldenrod">
                         <p><ctitle>OpenWhisk Client JS</ctitle>JavaScript client library for the OpenWhisk platform</p>
                         <div class="component-release-artifact-list border-darkgoldenrod">
-                          <version>3.21.4</version>
+                          <version>3.21.5</version>
```

Comment:
Seems 3.21.6 is available, should this be it?
https://github.com/apache/openwhisk-client-js/releases/tag/3.21.6

Reply:
I thought about it, but 3.21.6 was never committed to the Apache downloads directory.  I'm not sure why the release manager didn't finish the process. Since we're about to do 3.21.7, I wasn't that worried about it.

Reply:
got it.